examples/dedisperse_samples.py

```diff
@@ -16,43 +16,15 @@
 
 from timeseries.timeseries import Timeseries
 
-def print_possible_dm(data):
-        '''
-        This method is for testing purposes and should probably be removed later on
-        '''
-        first_sample = 400
-
-        for i, data_point in enumerate(data[first_sample]):
-                if(data_point > 10):
-                        print("first sample ", i, " - ", data_point)
-                        break
-
-        highest_difference = 0
-
-        for s, samples in enumerate(data):
-                for i, data_point in enumerate(samples):
-                        if(i > highest_difference):
-                                
-                                if(data_point > 10):
-                                        print(s, i, " - ", data_point)                                        
-                                        highest_difference = i
-                                break
-
-        print(highest_difference)
-
-
 # Read filterbank data
 special_pspm = fb.Filterbank(filename = "../data/my_special_pspm.fil")
```

Comment:
Change this to the required pspm file and let it sit in the example dir. All the other examples use ./pspm32.fil